### DISCUSSION TITLE
Can't see Google Search query data

### DISCUSSION THREAD

**Discussion by accentix** [plausible/analytics#2094] [Anything Else]
I have been adding sites to Plausible for several months now.  I'm up to 15. I was going to show a client how to access their search data today, but when I clicked on "Google" all I can see is a constantly circling icon. After combing through the docs and this forum, I checked the permissions at GSC. Everything looked fine. Next, I  delinked and then relinked a domain. Still nothing. 

I was hoping to standardize on Plausible for all of my clients, but since I'm typing my problem into a forum, I'm not feeling too comfortable about things. If this were some sort of free service, or if I was self-hosting, I would understand that I'm on my own. But, since I"m a paying customer, I kind of view the relationship as being like any other SAAS. I should have a method available where I can email support, or create a ticket. And, get a direct response in a timely manner. 

I can't rely on posting into the air when I'm trying to convince clients to dump GA, which, actually still works consistently. 

At any rate, does anyone have any insight as to why it's not working any longer and how to fix it?

**Comment by RobertJoonas** [comment]
Hi @accentix! We're very sorry for the inconvenience!

This seems to be a bug that came up with our recent changes to the Google Analytics import feature. This has already been fixed and merged ([PR](https://github.com/plausible/analytics/pull/2090)). We will release it today.

Thanks for your patience with this!
